src/neo4j_graphrag/experimental/pipeline/kg_builder.py

```diff
@@ -150,6 +162,18 @@ def __init__(
 
         self.pipeline = self._build_pipeline()
 
+    @staticmethod
+    def to_schema_entity(entity: EntityInputType) -> SchemaEntity:
+        if isinstance(entity, dict):
+            return SchemaEntity(**entity)
```

Comment:
Is there a case where `entity` cannot be converted to `SchemaEntity` (and `SchemaRelation` below)?

Reply:
You're right, it will happen if the dict has a wrong format. Do you think we should raise a specific error there? The default will be the Pydantic validation error.

Reply:
I think that is sufficient imo 😄